client/src/components/ReviewTab/ReviewTab.tsx

```diff
@@ -38,7 +72,36 @@ export default function PersonalInfo(): JSX.Element {
             <ReviewTabForm handleSubmit={handleSubmit} />
           </Grid>
         </Grid>
-      </Grid>
+      </Paper>
+      <Typography align="center" component="h1" variant="subtitle1" className={classes.readAll}>
+        Read all
+      </Typography>
+      <Switch checked={checked} onChange={handleChange} inputProps={{ 'aria-label': 'controlled' }} />
+      {checked === true
+        ? reviews &&
+          reviews.map((review) => (
+            <Box key={review._id} display="flex" justifyContent="center">
+              <Paper className={classes.paper}>
+                <Box display="flex" mb={2} textAlign="center" pt={2}>
+                  <Avatar alt="Profile Image" src={review.reviewerId.profilePic} className={classes.avatar}></Avatar>
+                  <Typography className={classes.username}>
+                    @{review.reviewerId.username}
+                  </Typography>
+                </Box>
+                <Box display="flex" mb={2} textAlign="center">
+                  <Rating value={review.rating} size="large" />
```

Comment:
Shouldn't review be a read only component here? :)

Reply:
it should be :) ., changing it